### PR TITLE
DOCS-513 Updates for free trial

### PR DESCRIPTION
**Pull request by angelasimms**
Closes https://hazelcast.atlassian.net/browse/DOCS-513

### CODE REVIEW

**Review by FinnLiuHZ — APPROVED** [review]

File: docs/modules/ROOT/pages/serverless-cluster.adoc [hunk 1]
```diff
@@ -83,25 +84,11 @@ Be aware that the following operations use heap memory: writing, caching, or rep
...
-- You don't connect to the cluster for 24 hours.
-- You don't have a payment method set up for your Hazelcast {hazelcast-cloud} account.
-
-If you're using a production cluster, Hazelcast saves your cluster data so that you can resume the cluster at a later time.
-
-If you're using a development cluster, all cluster data is lost after the cluster is paused.
-
 == Pricing for {hazelcast-cloud} Serverless Clusters
 
 You pay only for what you store in RAM. Any data that is stored on disk or in backups to disk is free of charge. As a result, 'storage' is synonymous with RAM in {hazelcast-cloud} Serverless clusters.
```
Comment (lovanchetty) [inline docs/modules/ROOT/pages/serverless-cluster.adoc:89]:
```suggestion
You pay for all artifacts that are stored. This includes data, metadata, custom code, etc. in {hazelcast-cloud} Serverless clusters.
```

File: docs/modules/ROOT/pages/serverless-cluster.adoc [hunk 2]
```diff
@@ -83,25 +84,11 @@ Be aware that the following operations use heap memory: writing, caching, or rep
 
 Additionally, Management Center has a heap memory limit of 3.7 GB shared between all clusters in an account. Running several streaming jobs with high workloads, or one uncontrolled streaming job, can lead to the temporary loss of metrics data.
```
Comment (lovanchetty) [inline docs/modules/ROOT/pages/serverless-cluster.adoc:85]: @FinnLiuHZ Why are we exposing this to users? We control MC and if it cannot handle a scenario we need to treat that as a product enhancement and evolve it to be better.

**Review by lovanchetty — APPROVED** [review]

File: docs/modules/ROOT/pages/overview.adoc [hunk 3]
```diff
@@ -27,7 +27,7 @@ Hazelcast {hazelcast-cloud} offers two editions: {hazelcast-cloud} Serverless an
 
 |xref:serverless-cluster.adoc[{hazelcast-cloud} Serverless]
 |{hazelcast-cloud} Serverless clusters auto-scale, depending on the amount of data you store.
-|Free to store up to 2 GiB in any {hazelcast-cloud} Serverless cluster in your account, then $0.10 USD per GiB.
+|$0.10 USD per GiB of data stored on any {hazelcast-cloud} Serverless cluster in your account.
 
 |xref:dedicated-cluster.adoc[{hazelcast-cloud} Dedicated] (coming soon)
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/overview.adoc:32]: Covered in a separate ticket.

File: docs/modules/ROOT/pages/overview.adoc [hunk 4]
```diff
@@ -27,7 +27,7 @@ Hazelcast {hazelcast-cloud} offers two editions: {hazelcast-cloud} Serverless an
...
 |{hazelcast-cloud} Dedicated clusters give you fine-grained control over the hardware and resources available to your cluster. It's your responsibility to scale {hazelcast-cloud} Dedicated clusters when you need more resources.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/overview.adoc:33]: Covered in a separate ticket.

File: docs/modules/ROOT/pages/serverless-cluster.adoc [hunk 5]
```diff
@@ -83,25 +84,11 @@ Be aware that the following operations use heap memory: writing, caching, or rep
...
-If you're using a production cluster, Hazelcast saves your cluster data so that you can resume the cluster at a later time.
-
-If you're using a development cluster, all cluster data is lost after the cluster is paused.
-
 == Pricing for {hazelcast-cloud} Serverless Clusters
 
 You pay only for what you store in RAM. Any data that is stored on disk or in backups to disk is free of charge. As a result, 'storage' is synonymous with RAM in {hazelcast-cloud} Serverless clusters.
 
-{hazelcast-cloud} Serverless clusters are billed monthly only for the amount of storage that you use. The *first 2 GiB of storage in your account is free*. If you use more than 2 GiB across all your clusters, you are charged $0.10 per hour for every additional GiB of used storage, not including the first 2 GiB.
+{hazelcast-cloud} Serverless clusters are billed monthly only for the amount of storage that you use. You are charged $0.10 per hour for every GiB of used storage.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/serverless-cluster.adoc:91]: Covered in a separate ticket.

File: docs/modules/ROOT/pages/serverless-cluster.adoc (hunk 2 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/serverless-cluster.adoc:85]: Covered in a separate ticket.

File: docs/modules/ROOT/pages/payment-methods.adoc [hunk 6]
```diff
@@ -1,22 +1,19 @@
...
 
 {description}
 
-For xref:serverless-cluster.adoc[{hazelcast-cloud} Serverless clusters], you must provide a payment card to do the following:
-
-- Create two or more {hazelcast-cloud} Serverless clusters.
-- Use more than 2 GiB of memory in a {hazelcast-cloud} Serverless cluster.
+For xref:serverless-cluster.adoc[{hazelcast-cloud} Serverless clusters], you must provide a payment card to create new clusters.
 
 For xref:dedicated-cluster.adoc[{hazelcast-cloud} Dedicated clusters], you must provide a payment card to pay for new clusters or cluster usage for existing clusters.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/payment-methods.adoc:12]: Covered in a separate ticket.

File: docs/modules/ROOT/pages/serverless-cluster.adoc (hunk 1 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/serverless-cluster.adoc:89]: Covered in a separate ticket.

File: docs/modules/ROOT/pages/free-trial.adoc [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+You can get support from our team at any point during your trial, either from within the trial environment or by sending an email to mailto:viridian@hazelcast.com[].
+
+You'll also receive emails from Hazelcast with useful resources to help you get up and running with your first cluster.
+
+== What Happens at the End of the Trial?
+
+At the end of the 14-day trial period, two things happen:
+
+- Your account is automatically suspended. This means that although you can still access the trial environment, your cluster is stopped and cannot be restarted.
+- Your trial environment will be deleted, including all data that you created during the trial.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/free-trial.adoc:32]: Updated.

File: docs/modules/ROOT/pages/get-started.adoc [hunk 8]
```diff
@@ -17,7 +17,7 @@ You need the following:
 == Step 1. Start a {hazelcast-cloud} Serverless Development Cluster
 
 include::serverless-cluster.adoc[tag=development]
-To create a development cluster, do the following:
+To create a development cluster, do the following. Or, if you already have a development cluster, jump straight to step 2.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/get-started.adoc:20]: Created two sections for this. One for the trial user and another for the paying customer.

File: docs/modules/ROOT/pages/serverless-cluster.adoc (hunk 1 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/serverless-cluster.adoc:89]: Updated as requested.

File: docs/modules/ROOT/pages/free-trial.adoc [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+You can get support from our team at any point during your trial by sending an email to mailto:viridian@hazelcast.com[].
+
+You'll also receive emails from Hazelcast with useful resources to help you get up and running with your first cluster.
+
+== What Happens at the End of the Trial?
+
+At the end of the 14-day trial period, two things happen:
+
+- Your account is automatically suspended. This means that although you can still access the trial environment, your cluster is stopped and cannot be restarted.
+- Your trial environment, including all data, will be automatically deleted.
```
Comment (alaktionov) [inline docs/modules/ROOT/pages/free-trial.adoc:32]: Cluster will be deleted after 7 days of trial expiration, would it make sense to add here?

File: docs/modules/ROOT/pages/serverless-cluster.adoc [hunk 10]
```diff
@@ -34,8 +34,9 @@ Development clusters have the following limitations:
 - No cluster scaling
 - No data backups
 - No cluster data persistence
+- Paused after 6 hours without development activity
```
Comment (alaktionov) [inline docs/modules/ROOT/pages/serverless-cluster.adoc:37]: Perhaps this is not necessary. Both development and production clusters are idled after 24 hours if there is no payment method.

File: docs/modules/ROOT/pages/free-trial.adoc (hunk 9 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/free-trial.adoc]: Updated as requested.

File: docs/modules/ROOT/pages/serverless-cluster.adoc (hunk 10 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/serverless-cluster.adoc]: Updated as requested.

**Review by alaktionov — APPROVED** [review]

**Pull request merged**